============================= test session starts ==============================
platform linux -- Python 3.5.2, pytest-3.2.5, py-1.5.2, pluggy-0.4.0
rootdir: /home/sklump/von_agent, inifile:
plugins: asyncio-0.8.0

========================= no tests ran in 0.00 seconds =========================
ERROR: file not found: 2
